src/Common/Core/Impl/UI/Commands/CommandStatus.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.R.Components.Controller {
+namespace Microsoft.Common.Core.UI.Commands {
```

Comment:
`IAsyncCommand` has `Status` property of `CommandStatus` enum, which has `Invisible` and `Latched` values that are UI-related.